.gitignore

```diff
@@ -1,9 +1,4 @@
-*.swp
-*.beam
-.DS_Store
```

Comment:
Right, it *should* be, but often it is not, especially when dealing with newbies. Since a lot of newbies get involved here, I'd rather avoid having to ask them to remove files, squash commits, and push force to their branch.

On the other hand, the right thing to do is to coach them in the right way to do it, so maybe it's worth it.